event_server.cpp

```diff
@@ -1861,6 +1861,36 @@ static void get_settling(JObj& response, const json_value *params)
     response << jrpc_result(settling);
 }
 
+static void get_variable_delay_settings(JObj& response, const json_value* params)
+{
+    JObj rslt;
+
+    VarDelayCfg delayParams = pFrame->GetVariableDelayConfig();
+    rslt << NV("Enabled", delayParams.enabled)
+        << NV("ShortDelay", delayParams.shortDelay / 1000)
+        << NV("LongDelay", delayParams.longDelay / 1000);
+    response << jrpc_result(rslt);
+}
+
+static void set_variable_delay_settings(JObj& response, const json_value* params)
+{
+    Params p("Enabled", "ShortDelay", "LongDelay", params);
+    const json_value* p0 = p.param("Enabled");
+    const json_value* p1 = p.param("ShortDelay");
+    const json_value* p2 = p.param("LongDelay");
+    bool enabled;
+    double shortDelay;
+    double longDelay;
+    if (!p0 || !p1 || !p2 || !bool_param(p0, &enabled) || !float_param(p1, &shortDelay) || !float_param(p2, &longDelay))
+    {
+        response << jrpc_error(JSONRPC_INVALID_PARAMS, "expected Enabled, ShortDelay, LongDelay params)");
+        return;
+    }
+    VarDelayCfg currParams;
+    pFrame->SetVariableDelayConfig(enabled, (int)shortDelay * 1000, (int)longDelay * 1000);

```

Comment:
better do the multiplication before the cast to int!
```suggestion
    pFrame->SetVariableDelayConfig(enabled, (int)(shortDelay * 1000.), (int)(longDelay * 1000.));
```

Reply:
Thanks.  I've posted the update, looking for another review.